cmd/query/app/querysvc/adjuster/warning.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package adjuster
+
+import (
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+)
+
+const (
+	adjusterWarningAttribute = "jaeger.adjuster.warning"
```

Comment:
```suggestion
	warningsAttribute = "jaeger.internal.warnings"
```
we need this as a general mechanism for warnings, not specific to adjusters. The UI will only look in one place for these warnings.

BTW since we're adding this as Span attribute we need to alter otlp->model translation to look for this attribute and move it to the proper Span.Warnings place. I think we should find all instances of us calling OTLP<>model translations in the code and replace them with internal methods so that we always go through the same logic (i.e. call transformer from OTEL first and then run our additional transformers).

Reply:
sounds good - should this attribute constant still live in package `adjuster`?

Reply:
I added an item for the otlp->model issue to the tracking issue to address in a follow-up PR

Reply:
no it should not be in the adjuster, we need a neutral place across all modules. Maybe internal/model_v2

Reply:
@yurishkuro do we want to introduce the name `model` for v2? it seems like model has been synonymous with v1

Reply:
I know, I don't like it either. Any other ideas?  `jotlp`?